src/payment/strategies/braintree/braintree-sdk-creator.ts

```diff
@@ -79,6 +79,7 @@ export default class BraintreeSDKCreator {
                     this._paypalcheckoutInstance = instance;
                     instance.loadPayPalSDK({
                         currency: config.currency,
+                        merchantId: config.merchantId,
```

Comment:
`config.merchantId` - this value could be null. Will SDK script initialised correctly for such case?